.github/workflows/validate_pr.yml

```
@@ -26,10 +26,10 @@ jobs:
     - name: Build with Maven
       run: |
         ./mvnw verify -U -Dmaven.javadoc.skip=true -V -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-    - name: Set up JDK 17 for SonarCloud
+    - name: Set up JDK 21 for SonarCloud
       uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)